service/arm_templates/main.json

```diff
@@ -0,0 +1,125 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterID": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique ID of this guest cluster."
+      }
+    },
+    "storageAccountType": {
+      "type": "string",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS",
+        "Standard_ZRS",
+        "Premium_LRS"
+      ]
+    },
+    "virtualNetworkCidr": {
+      "type": "string",
+      "metadata": {
+        "description": "The main CIDR block reserved for this virtual network."
+      }
+    },
+    "masterSubnetCidr": {
+      "type": "string"
+    },
+    "workerSubnetCidr": {
+      "type": "string"
+    },
+    "apiLoadBalancerCidr": {
+      "type": "string"
+    },
+    "etcdLoadBalancerCidr": {
+      "type": "string"
+    },
+    "ingressLoadBalancerCidr": {
+      "type": "string"
+    },
+    "kubernetesAPISecurePort": {
+      "type": "string"
```

Comment:
I've got another query. In `clustertpr` we store the ports as ints. So we need to convert to a string in the operator. WDYT about making these int parameters?